.github/CONTRIBUTING.md

```diff
@@ -153,6 +153,8 @@ example that compiles and passes [documentation tests](https://doc.rust-lang.org
 
 - All code should be well tested, passing both locally and in our [testing continuous integration](https://github.com/iotaledger/firefly/blob/develop/.github/workflows/ci.test.yml) workflow
 
+- The repository should be secure, passing our [Snyk continuous integration](https://github.com/iotaledger/firefly/blob/develop/.github/workflows/ci.snyk.yml) workflow (checking for security vulnerabilities)
```

Comment:
For PRs from forks that add dependencies, we may need to manually test them locally or merge them to a branch in this repo since `SNYK_TOKEN` isn't available to PRs from forks

Reply:
I suppose that once a forked PR is approved then we could merge it into a `snyk/your-branch` branch for just CI testing then if it passes merge to `develop`? Would only be if dependencies got added like you said

Reply:
Yep, that's what I was thinking

Reply:
I'm thinking there won't be enough cases of this to go to the effort of automating this process somehow... are we okay with doing it manually?

Reply:
I personally am since I don't anticipate most PRs from forks to add/change dependencies, but I'm happy to discuss automating it if you want

Reply:
Nice, same here if needed! We can cross the automation bridge another day ~